.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Should I file this?**
+Use this template to report *observed bugs in current Plotman*.  Do not use this to request tech support or troubleshooting; we are unable to provide that support here, so please use the user channel(s) on Keybase, or GitHub Discussions instead.  Do not use this to report issues with the core Chia plotter.  If you are using `main` branch, try pulling `development` to see if your problem is already fixed.
```

Comment:
We no longer recommend `git clone` as part of installation so 'pulling `development`' is not quite right.  I think I usually word it as 'Try the development branch by using `@development` in the readme installation instructions instead of `@main`'.  Though the readme ought to grow a mention of this as well.

Reply:
OK, addressed.